### PR TITLE
V6.18.00

### PR DESCRIPTION
**Pull request by JohanMabille**
@SylvainCorlay @chrisburr I'm not sure if the build number is correct.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,4 +1,2 @@
 clang_variant:
-  - default
-  - cling_6.14.06
-  - cling_6.16.00
+  - cling_6.18.00
```
Comment (chrisburr) [inline recipe/conda_build_config.yaml]: I think the default variant should be kept.
```suggestion
  - default
  - cling_6.18.00
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% set version = "5.0.0" %}
 {% set sha256 = "019f23c2192df793ac746595e94a403908749f8e0c484b403476d2611dd20970" %}
 
-{% set build_number = "1004" %}
+{% set build_number = "5" %}
```
Comment (chrisburr) [inline recipe/meta.yaml]: As the version is the same it's safer to increment it:
```suggestion
{% set build_number = "1005" %}
```

(As it's a new variant that will be used when pinned it's probably safe to use a lower one, but there is no benefit from risking it and will only cause confusion.)

File: .appveyor.yml [hunk 3]
```diff
@@ -10,13 +10,10 @@ environment:
     secure: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 
   matrix:
-    - CONFIG: win_clang_variantcling_6.14.06cxx_compilervs2015vc14
+    - CONFIG: win_cxx_compilervs2008
```
Comment (chrisburr) [inline .appveyor.yml]: I think you should remove this part of the matrix as well as `.ci_support/win_cxx_compilervs2008.yaml` as they're not supposed to be built according to the skip.

(due to the ugly hack I suggested on Gitter.)

File: .azure-pipelines/azure-pipelines-win.yml [hunk 4]
```diff
@@ -6,20 +6,24 @@ jobs:
...
         UPLOAD_PACKAGES: False
-      win_clang_variantcling_6.16.00cxx_compilervs2015vc14:
-        CONFIG: win_clang_variantcling_6.16.00cxx_compilervs2015vc14
+      win_clang_variantcling_6.18.00cxx_compilervs2015:
+        CONFIG: win_clang_variantcling_6.18.00cxx_compilervs2015
         CONDA_BLD_PATH: D:\\bld\\
         UPLOAD_PACKAGES: False
-      win_clang_variantdefaultcxx_compilervs2015vc14:
-        CONFIG: win_clang_variantdefaultcxx_compilervs2015vc14
+      win_clang_variantdefaultcxx_compilervs2008:
```
Comment (chrisburr) [inline .azure-pipelines/azure-pipelines-win.yml]: These lines and the associated config file should be deleted.

**Comment by JohanMabille** [comment]
@chrisburr do you think this can be merged?

**Comment by chrisburr** [comment]
In my opinion it can, but I don't have the permissions to do it.

@conda-forge/clangdev Can someone take a look at this PR?

**Pull request merged**